eng/InstallRuntimes.proj

```diff
@@ -95,7 +96,21 @@
     <Exec Command="bash $(DotnetInstallScriptCmd) $(CommonInstallArgs) %(RuntimeTestVersions.ExtraInstallArgs) -Version %(RuntimeTestVersions.AspNetDownload) -Runtime aspnetcore"
           IgnoreStandardErrorWarningFormat="true"
           Condition="'%(RuntimeTestVersions.AspNetDownload)' != ''" />
-   </Target>
+  </Target>
+
+  <!-- Overrides the latest runtime shared framework with local shared framework artifacts -->
+  <Target Name="OverrideLatestRuntime"
+          Condition="'$(LiveRuntimeDir)' != ''"
+          Inputs="$(VersionsPropsPath)"
+          Outputs="$(TestConfigFileName)">
+    
+    <ItemGroup>
+      <_LiveRuntimeFiles Include="$(LiveRuntimeDir)\**\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_LiveRuntimeFiles)"
```

Comment:
Most of the time this will work but there can be up to 3 versions of the latest (10) runtime in the .dotnet-test directory

1) The version installed with the test .NET 10 SDK
2) The version installed with the aspnetcore latest
3) The runtime version installed explicitly from the DARC update which is (the MicrosoftNETCoreAppRuntimewinx64Version property) that the $(DotNetInstallDir) is built from.

That is why I didn't automate the private build copying in the instructions. If you only copy #3, then most of the tests will run on that version except the couple of aspnetcore debuggee's (WebApp3, etc.).  For the couple of cdac specific tests you created this is sufficient, but for all the tests run to test the cdac private build, the aspnetcore runtime would also need to be copied.